package.json

```diff
@@ -5,8 +5,8 @@
   "main": "webpack.config.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "webpack serve --open",
-    "build": "webpack"
+    "build": "webpack --mode=development",
```

Comment:
- [ ] Please update the `build` script to be for production mode, and add another script attribute for the development mode called `dev` as required in this milestone. 🙏